.github/workflows/focus-ios-ui-tests-old.yml

```diff
@@ -0,0 +1,140 @@
+name: Focus iOS 15 & 16 tests
+on:
+  workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    test_results_directory: /Users/runner/tmp
+
+jobs:
+  Focus-iOS-Tests:
+    name: Focus iOS
+    runs-on: macos-14
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ios_version: 16.4
+            ios_simulator: iPhone 14
+          - ios_version: 15.5
+            ios_simulator: iPhone 13
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4.1.7
+        with:
+          repository: mozilla-mobile/firefox-ios
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Install packages
+        run: |
+          pip3 install virtualenv pipenv
+          gem install xcpretty -v 0.3.0
+          pip install blockkit==1.9.1
+          npm install -g junit-report-merger@7.0.0
+      - name: Setup Xcode
+        id: xcode
+        run: |
+          sudo rm -rf /Applications/Xcode.app
+          sudo rm -fr /Applications/Xcode_16*
+          sudo rm -fr /Applications/Xcode_14*
+          sudo xcode-select -s /Applications/Xcode_${{ env.xcode_version }}.app/Contents/Developer
+          xcodebuild -version
+          ./checkout.sh
+          ./bootstrap.sh --force
+      - name: Create iOS ${{ matrix.ios_version }} simulator
+        id: setup-simulator
+        run: |
+          sudo xcodes runtimes install "iOS ${{ matrix.ios_version }}"
```

Comment:
I discovered a more reliable way to install old runtime than before. Fastlane suggests `xcodes`.